app/src/main/java/org/fossasia/openevent/app/module/event/list/EventsPresenter.java

```diff
@@ -41,6 +46,15 @@ public void start() {
         return events;
     }
 
+    @Override
+    public void sortByName(int sortBy) {
```

Comment:
Really sorry for not mentioning it first, but this should be `sortBy(int criteria)` as this is not sorting by name